language/Russian/strings.po

```diff
@@ -2531,227 +2562,233 @@ msgid ""
 "--- DRIVER INFO ---\n"
 "Driver: "
 msgstr ""
+"\n"
+"\n"
+"--- ИНФО ДРАЙВЕРА ---\n"
+Драйвер: "
```

Comment:
There is a missing quote at the beginning of this line - that will break things.